packages/zowe-explorer/src/trees/uss/UssFSProvider.ts

```diff
@@ -133,7 +135,7 @@ export class UssFSProvider extends BaseProvider implements vscode.FileSystemProv
         }
         const response = await ZoweExplorerApiRegister.getUssApi(profile).fileList(ussPath);
         // If request was successful, create directories for the path if it doesn't exist
-        if (!this.exists(uri)) {
+        if (response.success && !(keepRelative ?? false) && response.apiResponse.items?.[0]?.mode?.startsWith("d") && !this.exists(uri)) {
```

Comment:
Curious why we don't just default the keepRelative parameter to false in the function signature here 😋 
Maybe that way we avoid a few operators 😋

Reply:
I agree, writing the fallback to false felt weird even though it has a similar effect to the default parameter value 😄
I think it makes sense to fix this here for developer clarity - resolved in commit fd904aba